packages/accordions/src/elements/components/StepContent.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useEffect, HTMLAttributes } from 'react';
+import { useCombinedRefs } from '@zendeskgarden/container-utilities';
+import { StyledContent, StyledInnerContent } from '../../styled';
+import { useStepContext, useStepperContext } from '../../utils';
+
+export const StepContent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
+    const contentRef = useCombinedRefs<HTMLDivElement>(ref);
+    const { activeIndex, isHorizontal } = useStepperContext();
+    const { currentStepIndex } = useStepContext();
+    const isActive = currentStepIndex === activeIndex;
+
+    useEffect(() => {
+      if (contentRef.current && isHorizontal === false) {
+        const child = contentRef.current.children[0] as any;
+
+        child.style.height = `${child.scrollHeight}px`;
```

Comment:
Note: this `height` is set for animation purposes. This `height` is not updated when the window resizes at this time.